src/lib.rs

```diff
@@ -116,14 +116,6 @@ impl Error {
         Error::Sys(errno::EINVAL)
     }
 
-    /// Get the errno associated with this error
```

Comment:
This is a public API, so removing this needs to be mentioned in the CHANGELOG under the `Removed` section.